fdbclient/NativeAPI.actor.cpp

```diff
@@ -2682,16 +2684,50 @@ void Transaction::setOption( FDBTransactionOptions::Option option, Optional<Stri
 			}
 
 			if(trLogInfo) {
-				if(!trLogInfo->identifier.present()) {
+				if (trLogInfo->identifier.empty()) {
```

Comment:
Can we implement this option by calling the two new options in sequence?

Reply:
Yes.